pkg/scheduler/cache/cache.go

```diff
@@ -684,6 +685,10 @@ func (sc *SchedulerCache) Snapshot() *schedulingapi.ClusterInfo {
 		}
 
 		snapshot.Nodes[value.Name] = value.Clone()
+
+		if value.RevocableZone != "" {
+			snapshot.RevocableNodes[value.Name] = value.Clone()
```

Comment:
`RevolcanoNodes` should be an index, so just one copy of `value.Clone()` should be enough.

Reply:
Fixed it.